templates/create_movies_form.html

```diff
@@ -1,9 +1,106 @@
+<!-- Links below used for help creating star rating system: -->
+<!-- https://gist.github.com/prof3ssorSt3v3/29e623d441e8174ffaef17741a1bba14 -->
+
 {% extends '_layout.html' %}
 
 {% block title %}Movie Rating App | Create Movie Rating{% endblock %}
 
 {% block body %}
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
 <h1 class="mb-5">Create Movie Rating</h1>
 <p class="mb-3">Create a movie rating below</p>
 <!-- TODO: Feature 2 -->
-{% endblock %}
+<div style="margin-top: 30px;">
+    <form action="/movies" method="get">
+        <div>
+            <label for="title">Enter a movie title:</label><br>
+            <input type="text" name="title" id="title" autocomplete="off" size="180px"><br>
+        </div>
```

Comment:
For this form, you could apply the bootstrap classes to give it better, more uniform styling
https://getbootstrap.com/docs/4.0/components/forms/